src/NuGet.Core/NuGet.Build.Tasks.Pack/Strings.resx

```diff
@@ -163,7 +163,7 @@
     <value>Invalid target framework for the file '{0}'.</value>
   </data>
   <data name="IsPackableFalseError" xml:space="preserve">
-    <value>Cannot generate a package from a project which has IsPackable property set to false.</value>
+    <value>Skipping package generation from this project because packaging has been disabled. Add &lt;IsPackable&gt;true&lt;/IsPackable&gt; to the project file to allow producing a package from this project.</value>
```

Comment:
It'd be nice if you generate the designer in the same change, that way it doesn't cause crazy diffs when others make changes to this resource.

Reply:
I have also updated the PR with a tracking issue, we try to have each PR associated with an issue.